assets/constants.ts

```diff
@@ -5072,6 +5072,9 @@ export default {
     "Geometry": "#be5637",
     "Trigonometry": "#be5637",
 
+    "Science": "#6f7a48",
```

Comment:
@HardikGoyal2003 There is already an ordering to this file. Please group "Science" with the science subjects (put it after "Physics" since they're in alphabetical order) and "Finance" with the soft-science subjects (put it between Economics/Geography). Thanks!

(In general, try to understand the existing structure and preserve it where possible, don't just put new stuff wherever. That just causes things to get messy over time.)

Reply:
Done, I will remember that, Thanks!